moped-editor/src/views/projects/projectView/ProjectSummary/SubprojectsTable.js

```diff
@@ -280,33 +325,41 @@ const SubprojectsTable = ({
     handleDeleteOpen,
     handleSaveClick,
     handleCancelClick,
+    deleteIcon: <LinkOffIcon sx={{ fontSize: 24 }} />,
   });
 
   if (loading || !data) return <CircularProgress />;
 
+  const noSubprojectsLabel = isSubproject
+    ? "Projects with a parent cannot have their own subprojects"
+    : "No subprojects to display";
+
   return (
     <>
       <DataGridPro
         sx={{ ...dataGridProStyleOverrides, border: 0 }}
         columns={dataGridColumns}
         rows={rows}
         autoHeight
+        getRowHeight={() => "auto"}
```

Comment:
@mateoclarke thanks for adding this in! Could  you add `density="comfortable"` too? Since we use these two parameters together in other project tables and it keeps some spacing around the input like we have in prod.

This is a good reminder for me to create a tech debt issue for us to abstract shared config in these tables.